lib/phoenix/template/eex_engine.ex

```diff
@@ -10,7 +10,7 @@ defmodule Phoenix.Template.EExEngine do
   end
 
   defp engine_for(name) do
-    case Phoenix.Template.format_encoder(name) do
+    case Phoenix.Template.format_encoder(Mix.Phoenix.otp_app(), name) do
```

Comment:
We shouldn't access the mix config in our app code. Mix won't be available in prod when using releases.